test/org/zalando/stups/mint/worker/external/etcd_test.clj

```diff
@@ -0,0 +1,22 @@
+(ns org.zalando.stups.mint.worker.external.etcd-test
```

Comment:
Additional tests for `run` would be cool :) https://github.com/zalando-stups/mint-worker/blob/master/test/org/zalando/stups/mint/worker/job/run_test.clj